src/classes/TDTM_DefaultConfig.cls

```diff
@@ -37,6 +37,36 @@
                settings and the new ones.
 */
 public without sharing class TDTM_DefaultConfig {
+    
+    public static boolean ADDR_Address_TDTM_TestAsync = false;
+    private static boolean ADDR_Address_TDTM_TestAsyncTemp;
```

Comment:
when testing a given trigger handler, you have to manually modify this file to set the value to true.  if we pursue an automated test strategy, then we will need to provide some way to set these flags from some config some way.